src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {post.user !== undefined && <UserInfo user={post.user} />}
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    {post.comments.length > 0 ? (
+      <CommentList comments={post.comments} />
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
And here you forgot the `<hr />` tag, so you dont have a line before **No comments yet** message.

Reply:
Please handle this